specification/paths/ServiceRates-service_rate_id.json

```diff
@@ -44,7 +44,34 @@
                     {
                       "required": [
                         "id"
-                      ]
+                      ],
+                      "properties": {
+                        "relationships": {
+                          "properties": {
+                            "service_options": {
+                              "properties": {
+                                "data": {
+                                  "items": {
```

Comment:
Don't we miss a "type": "array", above here? Or don't we need this because of inheritance.

Reply:
Inheritance